dtm-database-backend/src/main.rs

```diff
@@ -19,6 +64,21 @@ async fn main() {
     .unwrap();
 }
 
+/// A booting mode getter for this server which is hyper.
+/// this function is not pure, and write out message to stdout.
+fn get_booting_mode(mode_env: &str) -> BootingModes {
+  match std::env::var(mode_env) {
+    Ok(value) => match value.parse::<BootingModes>().unwrap() {
+      BootingModes::Debug => BootingModes::Debug,
+      BootingModes::Release => BootingModes::Release,
+    },
```

Comment:
ここって一回match噛ませてる理由ありますか……？
```suggestion
    Ok(value) => value.parse::<BootingModes>().unwrap()
```